Source/Array.swift

```diff
@@ -57,25 +57,25 @@ public func -<<<T, U>(@noescape f: T -> [U], a: [T]) -> [U] {
 /**
     Wrap a value in a minimal context of []
 
-    :param: a A value of type T
+    - parameter a: A value of type T
 
-    :returns: The provided value wrapped in an array
+    - returns: The provided value wrapped in an array
 */
 public func pure<T>(a: T) -> [T] {
     return [a]
 }
 
-extension Array {
+public extension Array {
     /**
         apply an array of functions to self
 
         This will return a new array resulting from the matrix of each function being applied to each value inside self
 
-        :param: fs An array of transformation functions from type T to type U
+        - parameter fs: An array of transformation functions from type T to type U
 
-        :returns: A value of type [U]
+        - returns: A value of type [U]
     */
-    func apply<U>(fs: [T -> U]) -> [U] {
+    func apply<U>(fs: [Element -> U]) -> [U] {
```

Comment:
I actually prefer `T` to `Element`, did it change because there's no way to indicate the type of the elements in an Array extension?